src/forms/_base.scss

```diff
@@ -345,17 +365,17 @@ select {
 }
 
 .form-help {
-  @include chronicle-italic;
-
-  font-size: 14px;
-  color: color(text);
-  margin-top: 0.7rem;
-  font-style: italic;
-  letter-spacing: size(smallest-tracking);
   display: block;
+  font-size: ms(-1);
+  color: color(primary);
+  margin-top: 0.7rem;
 
   a {
-    @include link(text, medium-text, "after");
+    @include link(text, dark-text, "after");
+  }
+
+  &.form-help--tight {
```

Comment:
Not sure, this was carried over from casper repo, prob where there needed to be less margin in some instance.